_includes/manuals/2.4/3.6_upgrade.md

```diff
@@ -147,7 +193,13 @@ You should clear the cache and the existing sessions:
     foreman-rake tmp:cache:clear
     foreman-rake db:sessions:clear
 
-##### Optional Step 3 (B) - Run foreman-installer
+##### Optional Step 3 (B) - Reclaim database space
+
+After database migrations, some space can sometimes be reclaimed. It's a good idea to perform a *full* database vacuum for PostgreSQL rather than relying on the autovacuum feature to claim maximum space possible.
+
+    su - postgres -c 'vacuumdb --full --dbname=foreman'
+
+##### Optional Step 3 (C) - Run foreman-installer
```

Comment:
considering we are aiming to have the installer as the main recommended way, shouldn't this come first and not marked as optional (with a comment it isn't relevant if you aren't using the installer)

Reply:
I didn't want to got that far. Still plenty of improvement possible but this is now clearer than it used to be (IMHO).